components/sysutils/tmux/Makefile

```diff
@@ -11,20 +11,21 @@
 #
 # Copyright 2011, 2013, EveryCity Ltd. All rights reserved.
 # Copyright 2019, Michal Nowak
+# Copyright 2019, Solene Rapenne
 #
 
```

Comment:
Please try to adapt to the new style (see runtime/node12 as an example):
Insert

BUILD_BITS=    64

before importing shared-macros.mk

Later replace
include $(WS_MAKE_RULES)/prep.mk
include $(WS_MAKE_RULES)/configure.mk
include $(WS_MAKE_RULES)/ips.mk

with
TEST_TARGET=    $(NO_TESTS)
include $(WS_MAKE_RULES)/common.mk

and remove

build:		$(BUILD_64)
install:	$(INSTALL_64)
test:		$(NO_TESTS)